src/main/java/org/jetbrains/research/anticopypaster/models/UserSettingsModel.java

```diff
@@ -58,25 +68,64 @@ public void setComplexitySensitivity(int sensitivity){
     }
 
     public void setSizeSensitivity(int sensitivity){
+        System.out.println("We are changing the size sensitivity");
         this.sizeSensitivity = sensitivity;
         this.sizeMetrics.changeSensitivity(sensitivity);
     }    
 
     /**
-    Currently stubbed out to default to medium sens
-    This should ideally read in whatever the sensitivities are from the frontend,
-    the frontend 
+    Defaulted to medium if the user has not set up flag values,reads in
+     whatever the sensitivities are from the frontend if the user has set values
     */
     private void readSensitivitiesFromFrontend(){
+        //Default values if the user has not yet specified flag values
+        int keywordsSensFromFrontend = 2;
         int sizeSensFromFrontend = 2;
         int complexitySensFromFrontend = 2;
-        int keywordsSensFromFrontend = 2;
 
-        setComplexitySensitivity(complexitySensFromFrontend);
-        setSizeSensitivity(sizeSensFromFrontend);
+        //Grabs the sensitivity of the flags from the file if the file exists
+        File file = new File(FILE_PATH);
+        if (file.exists()) {
+            try (Scanner scanner = new Scanner(file)) {
+                //throw away first line
+                scanner.nextLine();
+                String keywordsDropdownValue = scanner.nextLine();
+                String keywordsCheckboxValue = scanner.nextLine();
+                String sizeDropdownValue = scanner.nextLine();
+                String sizeCheckboxValue = scanner.nextLine();
+                String complexityDropdownValue = scanner.nextLine();
+                String complexityCheckboxValue = scanner.nextLine();
+
+                keywordsSensFromFrontend = sensitivityInteger(keywordsDropdownValue);
+                sizeSensFromFrontend = sensitivityInteger(sizeDropdownValue);
+                complexitySensFromFrontend = sensitivityInteger(complexityDropdownValue);
+            } catch (FileNotFoundException ex) {
+                // Handle file not found exception
```

Comment:
We probably want to have at least a comment or a print here, just to have something to acknowledge that it was caught, more for readability for people using it in the future than anything else